src/index.css

```diff
@@ -1,10 +1,9 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: #1D2C3C;
```

Comment:
Isn't this color in a variable?

Reply:
Can we take the color from a tsx file to put it in a css/styled ?

Reply:
Yeah, it's madrugada but I can't import JS variables in a CSS files

Reply:
> Can we take the color from a tsx file to put it in a css/styled ?

What do you mean?

Reply:
> Yeah, it's madrugada but I can't import JS variables in a CSS files
This is the answer to my quetion. Thanks @Gpx !